test/merkle_patricia_tree_test.exs

```diff
@@ -1,4 +1,60 @@
 defmodule MerklePatriciaTreeTest do
   use ExUnit.Case
 
+  alias MerklePatriciaTree.Trie
+
+  @passing_tests %{
+    anyorder: [
+      :singleItem,
+      :dogs
+    ]
+  }
+
+  test "Ethereum Common Tests" do
```

Comment:
I like your approach to integrating with ethereum tests.

Reply:
It's from @masonforest :)